core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -769,19 +781,27 @@ object ConsumerGroupCommand extends Logging {
       }
     }
 
-    private def checkOffsetRange(topicPartition: TopicPartition, offset: Long) = {
-      getLogEndOffset(topicPartition) match {
-        case LogOffsetResult.LogOffset(endOffset) if offset > endOffset =>
-          warn(s"New offset ($offset) is higher than latest offset. Value will be set to $endOffset")
-          endOffset
+    private def checkOffsetRange(requestedOffsets: Map[TopicPartition, Long]) = {
```

Comment:
Nit: perhaps we should change the method name to checkOffsetsRange(), with plural form.

Reply:
Sure, will update this too.